app/presenters/downstream_presenter.rb

```diff
@@ -47,11 +47,19 @@ def links
 
     def access_limited
       return {} unless access_limit
-      {
-        access_limited: {
-          users: access_limit.users
+      if web_content_item.state == 'published'
```

Comment:
It's possible for an item to be unpublished here as well, I think it'd be safer to use `state != 'draft'`